AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -119,6 +119,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+pcking.de###es_cookie_solution
```

Comment:
It looks like we can block it as a request `||pcking.de/_javascripts/cookiesolution/`